src/latexify/frontend.py

```diff
@@ -10,18 +10,10 @@
 from latexify import exceptions
 from latexify import parser
 from latexify import transformers
+from latexify.config import Config
 
 
-def get_latex(
-    fn: Callable[..., Any],
-    *,
-    identifiers: dict[str, str] | None = None,
-    reduce_assignments: bool = False,
-    use_math_symbols: bool = False,
-    use_raw_function_name: bool = False,
-    use_signature: bool = True,
-    use_set_symbols: bool = False,
-) -> str:
+def get_latex(fn: Callable[..., Any], *, config: Config | None = None, **kwargs) -> str:
     """Obtains LaTeX description from the function's source.
 
     Args:
```

Comment:
Add appropriate "Args" section.

I also think that this section should have only descriptions about `config` and `**kwargs`. Descriptions of specific configs can be written in the `Config` class instead.